16_woman_cancer.do

```diff
@@ -55,6 +59,7 @@ if _rc==0 {
 	replace w_papsmear=. if s714dd==9 | s714ee==9
 }
 
+/*Lesotho2009 RW Comment: s1012b/c are for Diabetes. The section below is problematic*/
```

Comment:
Nice catch! The updated template has this section deleted in the latest version.